docs/index.md

```diff
@@ -16,7 +14,7 @@ to understand this technical representation better.
 ## FIWARE representation
 
 The common technical representation of most of the data models is 
-[FIWARE NGSI](https://fiware-datamodels.readthedocs.io/en/latest/howto/index.html).  
+[FIWARE NGSI-LD](https://fiware-datamodels.readthedocs.io/en/latest/ngsi-ld_howto/index.html).  
 
```

Comment:
Adding also this line might help:
[FIWARE NGSI-LD tutorials](https://github.com/FIWARE/tutorials.Linked-Data)

Reply:
Thanks, added.